openmetadata-ui/src/main/resources/ui/src/pages/SearchIndexDetailsPage/SearchIndexDetailsPage.tsx

```diff
@@ -346,80 +345,75 @@ function SearchIndexDetailsPage() {
   const onExtensionUpdate = useCallback(
     async (updatedData: SearchIndex) => {
       searchIndexDetails &&
-        (await saveUpdatedSearchIndexData({
-          ...searchIndexDetails,
-          extension: updatedData.extension,
-        }));
+        (await onSearchIndexUpdate(
+          {
+            ...searchIndexDetails,
+            extension: updatedData.extension,
+          },
+          'extension'
+        ));
```

Comment:
Don't we need to update the state post patch call here?

Reply:
This is being handled by the onSearchIndexUpdate function